test/hotspot/jtreg/compiler/intrinsics/string/TestStringIntrinsics2.java

```diff
@@ -43,6 +43,28 @@
  *        compiler.intrinsics.string.TestStringIntrinsics2
  */
 
+/*
+ * @test id=LargeHeap
```

Comment:
Is it useful to run all other tests in the TestStringIntrinsics2 using the large heap?